XiEditor.xcodeproj/xcshareddata/xcschemes/XiEditor.xcscheme

```diff
@@ -39,7 +40,7 @@
             </BuildableReference>
          </TestableReference>
          <TestableReference
-            skipped = "NO">
```

Comment:
does this also disable locally? One option might be to have a separate scheme for CI?

Reply:
This is one option. Other option could be removing UI Tests from the main scheme and creating a separate scheme just for UI tests.

Reply:
Sure, that makes sense. Either or. It's also probably not that important at this stage either way, just thinking out loud.

@rakaramos I'm happy to merge this; what's involved in correcting the CI badges? Do we need to do that separately?

Reply:
@cmyr I guess the repo is already configured on travis, correct? If so, I can change it on my PR.

As for the coverage, it's just matter of hooking it up with [codecov](https://codecov.io/). It should be easy.

Let me know if need any anything, I'll be glad to help!

Reply:
Okay, I've turned CI back on for https://travis-ci.com/xi-editor/xi-mac and added xi-mac to codecov at https://codecov.io/gh/xi-editor/xi-mac. Codecov gave me a token but maybe I don't need that for a public repo?

If you can change the relevant bits of this PR we can try it out. 👍

Reply:
yes, token is not necessary for public repos.

I've changed the links, should be fine now

Reply:
@rakaramos thoughts on 9.2?

Reply:
I can change that, sure!